com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -11,11 +11,50 @@
 using Unity.Multiplayer.Tools;
 #endif
 using Unity.Profiling;
+#if NETCODE_USE_ADDRESSABLES
+using UnityEngine.AddressableAssets;
+#endif
 using UnityEngine.SceneManagement;
 using Debug = UnityEngine.Debug;
 
 namespace Unity.Netcode
 {
+    public enum NetworkManagerState
```

Comment:
❤️ 
I almost did this in a previous PR but then restrained myself.
Migrating towards a state driven NetworkManager is always a good thing in my book!